transform/variants/variants.go

```diff
@@ -0,0 +1,84 @@
+package variants
+
+import (
+	"errors"
+	"strings"
+)
+
+// AllVariantsIUPAC takes a string as input
+// and returns all iupac variants as output
+func AllVariantsIUPAC(seq string) ([]string, error) {
+	seqVariantList := [][]rune{}
+	seqVariants := []string{}
```

Comment:
Did this remove contribution?

Reply:
It shouldn't have? I think historically it has done that but git seemed to be tracking where code was going. I'll look into it.